File: ch_gpio/ledblink.c
#include <stdio.h>
#include <wiringPi.h>

int main()
{
    wiringPiSetupPhys();
    pinMode(12, OUTPUT);

    while(1){
        digitalWrite(12, HIGH);
        delay(1000);
        digitalWrite(12, LOW);
        delay(1000);
    }

    return 0;
}

File: ch_gpio/ledblink.py
#!/usr/bin/env python

import time
import RPi.GPIO as GPIO

LED_PIN = 12
GPIO.setmode(GPIO.BOARD)
GPIO.setup(LED_PIN, GPIO.OUT)

while True:
	print("LED is on")
	GPIO.output(LED_PIN, GPIO.HIGH)
	time.sleep(1)
	print("LED is off")
	GPIO.output(LED_PIN, GPIO.LOW)
	time.sleep(1)

GPIO.cleanup()

